app/services/search/response_item.rb

```diff
@@ -0,0 +1,8 @@
+# Simple model to represent single item in search query response
+module Search
+  class ResponseItem
+    include ActiveModel::API
+    include ActiveModel::Attributes
+    attr_accessor :title, :media_url, :thumbnail_url, :media_type, :url, :text, :external_id
```

Comment:
might we not also need to access the 'source'?

Reply:
This simple poro is used to abstract response from source. So we will know exact source at this moment. In database we do have source column, but here I believe it would be redundant